### PR TITLE
Bn fabb80

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
Please add to the description the infor on ruling:  <ab type="ruling"> The upper line is written below the ruling.</ab><ab type="ruling"> The bottom line is written above the ruling.</ab>
I personally inclined to think that Conti Rossini's dating is too broad, he himself might have included only 16th century because of the form of "lo" which he found modern, but this form do occurs in the 15th century and before.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb80.xml [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+            <citedRange unit="number">133</citedRange>
+            </bibl>
+         <bibl>
+            <ptr target="bm:Chaine1912Catalogue"/>
+            <citedRange unit="page">53-54</citedRange>
+            <citedRange unit="number">80</citedRange>
+         </bibl>
+         <bibl>
+            <ptr target="bm:Abbadie1859Cat"/>
+            <citedRange unit="page">93-94</citedRange>
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply ParisBNF/abb/BNFabb80.xml:293]: I checked the pdf of d'Abbadie's catalogue and it seems correct.

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: ParisBNF/abb/BNFabb80.xml [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+  Foliation of <placeName ref="INS0303BNF">Bibliothèque
+  nationale de France</placeName> in the upper
+    right corner of the recto sides.
+  </foliation>
+  <condition key="deficient">Many folios are mutilated at the edges.</condition>
+  </supportDesc>
+
+  <layoutDesc>
+     <layout columns="2" writtenLines="28 30">
+     <ab type="ruling"> The upper line is written below the ruling.</ab>
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply ParisBNF/abb/BNFabb80.xml]: I hope I chose the correct ruling pattern, I don't have much experience with that, I'm afraid.

File: ParisBNF/abb/BNFabb80.xml [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+<decoNote xml:id="d3">
+<locus target="#3r #7r #25r #27v #32v"/>
+<desc>More elaborate decorative bands are at the beginning of <ref target="ms_i1.1">the introduction</ref> and the homilies
+<ref target="ms_i1.2">on the birth of Christ</ref>, <ref target="ms_i1.4">for the beginning of Lent</ref>,
+for the <ref target="ms_i1.5">first</ref> and the <ref target="ms_i1.7">third</ref> Sunday of Lent.</desc>
+</decoNote>
+
+
+</decoDesc>
+
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply ParisBNF/abb/BNFabb80.xml:248]: Done.

File: ParisBNF/abb/BNFabb80.xml [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+            <change who="MKr" when="2021-04-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+    <body>
+      <listBibl type="catalogue">
+         <bibl>
+            <ptr target="bm:ContiRossini1914Notice"/>
+            <citedRange unit="page">15-16</citedRange>
+            <citedRange unit="number">133</citedRange>
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply ParisBNF/abb/BNFabb80.xml:307]: I don't have this reprint, only the original from Journal Asiatique in parts.

File: ParisBNF/abb/BNFabb80.xml [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+<desc>More elaborate decorative bands are at the beginning of <ref target="ms_i1.1">the introduction</ref> and the homilies
+<ref target="ms_i1.2">on the birth of Christ</ref>, <ref target="ms_i1.4">for the beginning of Lent</ref>,
+for the <ref target="ms_i1.5">first</ref> and the <ref target="ms_i1.7">third</ref> Sunday of Lent.</desc>
+</decoNote>
+
+
+</decoDesc>
+
+</physDesc>
+
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply ParisBNF/abb/BNFabb80.xml:274]: Added <history>

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**